server/game/core/Player.js

```diff
@@ -674,69 +657,20 @@ class Player extends GameObject {
             adjuster.canAdjust(playingType, card, target, ignoreType, penaltyAspect)
         );
         var costIncreases = matchingAdjusters
-            .filter((a) => a.getAmount(card, this) < 0)
-            .reduce((cost, adjuster) => cost - adjuster.getAmount(card, this), 0);
+            .filter((adjuster) => adjuster.direction === CostAdjustDirection.Increase)
```

Comment:
It seems a bit silly to call out the adjustment direction with its own enum rather than just using positive and negative numbers. Do the rules treat increases and decreases differently or something?